Regions.py

```diff
@@ -5,84 +5,126 @@
-            'ECT 100 Rupee Chest', 'Delivery to Mama Kafei'],
-            ['Honey and Darling', 'Treasure Chest Shop', 'Town Shooting Gallery', 'East Mailbox',
-            'Milk Bar', 'Stock Pot Inn', 'Stock Pot Inn Secret Entrance', 'Mayors Office',
+            ['Free The Postman (?)', 'ECT 100 Rupee Chest'],
+            ['To Honey and Darling', 'To Treasure Chest Shop', 'To Town Shooting Gallery', 'East Mailbox',
+            'To Milk Bar', 'To Stock Pot Inn', 'To Stock Pot Inn Secret Entrance', 'To Mayors Office',
             'Bomber Bouncer', 'ECT Top Exit to SCT', 'ECT Bottom Exit to SCT',
             'ECT Exit to NCT', 'Clock Town East Gate']),
         create_interior_region('Honey and Darling',
             ['Honey and Darling Bombchu Bowling Prize', 'Honey and Darling Archery Prize',
             'Honey and Darling Basket Bomb Throw Prize', 'Honey and Darling Grand Champion'],
             ['Honey and Darling Exit']),
-        create_interior_region('Treasure Chest Shop', ['Treasure Chest Game Piece of Heart Prize'], ['Treasure Chest Shop Exit']),
-        create_interior_region('Town Shooting Gallery', ['Town Shooting Beat the Record', 'Town Shooting Perfect Score'], ['Town Shooting Gallery Exit']),
-        create_interior_region('Milk Bar', ['Bar Performance'], ['Milk Bar Exit']),
+        create_interior_region('Treasure Chest Shop',
+            ['Treasure Chest Game Goron Prize', 'Treasure Chest Game Human Prize',
+             'Treasure Chest Game Zora Prize', 'Treasure Chest Game Deku Prize'], ['Treasure Chest Shop Exit']),
+        create_interior_region('Town Shooting Gallery', ['Town Shooting Gallery Quiver Prize', 'Town Shooting Gallery HP Prize'], ['Town Shooting Gallery Exit']),
+        create_interior_region('Milk Bar', ['Milk Bar Performance', 'Delivery to Mama Kafei'], ['Milk Bar Exit']),
         create_interior_region('Stock Pot Inn',
-            ['Do You Have a Reservation?', 'Have you seen this man?',
-            'What is the Carnival of Time?', 'Who are the Four Giants?',
-            'Love in the Toilet?', 'Stock Pot Reservation Chest', 'Anjus Chest',
+            ['Stock Pot Inn Key', 'Have you seen this man? (?)',
+            'Grandma Stories HP 1', 'Grandma Stories HP 2',
+            'Toilet Hand HP', 'Your Room Rupee Chest', 'Anjus Room Rupee Chest',
             'We Shall Greet The Morning Together'],
             ['Stock Pot Inn Roof Exit', 'Stock Pot Inn Front Door']),
-        create_interior_region('Mayors Office', ['Expert Person Solver Takes the Case', 'Love is the True Mayor'], ['Mayors Office Exit']),
-        create_interior_region('Bomber Tunnel', ['Bomber Tunnel Chest'], ['Bomber Tunnel Exit', 'Tunnel Balloon From ECT']),
+        create_interior_region('Mayors Office', ['Expert Person Solver Takes the Case', 'Mayor HP'], ['Mayors Office Exit']),
+        create_interior_region('Bomber Tunnel', [], ['Bomber Tunnel Exit', 'Tunnel to Sewers']),

```

Comment:
True, in that direction, but I had asked about the goron form getting through this area, and apparently it can't get back from the sewers area: it won't drown in the water, but can't lift itself up that ledge. It might be a pointless check, even with entrance shuffle, but conceptually, it should probly be there.

Reply:
The only reason that would be in logic, is if the Goron could actually get in without having any other form, which I think it gets blocked from the AO side by the Balloon?